src/test/plugins/ucc_discovery_plugin_test.cpp

```diff
@@ -18,15 +22,83 @@
 #include "logical_query_plan/predicate_node.hpp"
 #include "logical_query_plan/stored_table_node.hpp"
 #include "logical_query_plan/union_node.hpp"
+#include "operators/delete.hpp"
 #include "operators/get_table.hpp"
+#include "operators/insert.hpp"
 #include "operators/projection.hpp"
 #include "operators/table_scan.hpp"
+#include "operators/table_wrapper.hpp"
 #include "operators/update.hpp"
 #include "operators/validate.hpp"
+#include "storage/constraints/constraint_utils.hpp"
+#include "storage/constraints/table_key_constraint.hpp"
 #include "storage/storage_manager.hpp"
 #include "storage/table.hpp"
+#include "storage/table_column_definition.hpp"
+#include "types.hpp"
 #include "utils/load_table.hpp"
 #include "utils/plugin_manager.hpp"
+#include "utils/template_type.hpp"
+
+namespace {
+using namespace hyrise;  // NOLINT(build/namespaces)
+
+void _encode_table(const std::shared_ptr<Table>& table, const SegmentEncodingSpec& encoding_spec) {
```

Comment:
Pick here: These functions are not some protected/private members. Remove the prefix.